src/app/app.component.spec.ts

```diff
@@ -32,6 +36,18 @@ describe('AppComponent', () => {
         activeAccount: undefined,
       }),
       MockProvider(Document),
+      {
+        provide: FeedbackFinToken,
+        useValue: {
+          config: {},
+        },
+      },
+      {
+        provide: LITTILCONFIG,
+        useValue: ({
+          apiHost: 'localhost',
```

Comment:
Dit zal alleen lokaal goed gaan. environment variabele gebruiken ?

Reply:
Dit is een test, dus deze configuratie lijkt me prima. De vraag is of dit adres ook daadwerkelijk gebruikt wordt in de test, maar ik denk dat het antwoord "nee" is.

Reply:
Ow, dat heb ik zelf geschreven. Een WC eendje! In ieder geval, ik heb net main binnen gemerged, dus ook deze test gaat nog een keertje draaien.